app/Module.scala

```diff
@@ -18,14 +25,20 @@ class Module(
     configuration: Configuration
 ) extends AbstractModule {
 
-  override def configure() = {
+  override def configure(): Unit = {
+    val underlyingConfig = configuration.underlying
+    val sbrCtrlUnitRepositoryConfig = SbrCtrlUnitRepositoryConfigLoader.load(underlyingConfig)
+    bind(classOf[SbrCtrlUnitRepositoryConfig]).toInstance(sbrCtrlUnitRepositoryConfig)
 
-    //    val config = SBRPropertiesConfiguration.envConfig(ConfigFactory.load())
-    //    //    val config: Config = ConfigFactory.load
-    //    bind(classOf[Config]).toInstance(config)
+    bind(new TypeLiteral[Reads[UnitLinks]]() {}).toInstance(UnitLinks.reads)
+    bind(new TypeLiteral[Writes[LinkedUnit]]() {}).toInstance(LinkedUnit.writes)
```

Comment:
It might be nice to have a comment with a link to documentation on the `TypeLiteral` syntax

Reply:
I'll look to see what I can find - I seem to recall it took quite a while to find how to do this orginally ....